packages/website/components/SidebarLink.tsx

```diff
@@ -129,9 +136,21 @@ export function SidebarLink({
             {children}
             {isExternal && <ExternalLinkTrimmedIcon variant="muted" />}
           </span>
-          {isNew && (
-            <Badge className={styles.badge} variant="primary">
-              new
+          {(isNew || isDeprecated || isBeta || isAlpha) && (
```

Comment:
Will we still use `new` in some case?? or could we remove it?

Reply:
We would use it for components like "Pagination". - stable but new.

Reply:
I think it is kinda still valuable to indicate new components, that are just got out of beta.